vault/packaging/postinst

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cp -a /opt/rudder/share/plugins/vault/usr/share/ncf/tree /usr/share/ncf/
```

Comment:
@ncharles do you think we should put it with CFEgine stubs for dsc methods instead?